internal/slice/slice.go

```diff
@@ -9,3 +9,13 @@ func Contains[E comparable](s []E, v E) bool {
 	}
 	return false
 }
+
+// ContainsAny reports whether val is present in values
+func ContainsAny(values []interface{}, val interface{}) bool {
```

Comment:
nit: code style: we might want to make the variable name consistant with `Contains()`.
```suggestion
func ContainsAny(s []any, v any) bool {
```